api/node/__test__/api.spec.ts

```diff
@@ -45,6 +45,48 @@ test('loadFile', (t) => {
     ]);
 })
 
+test('loadSource', (t) => {
+    const source = `export component Test {
+        out property <string> check: "Test";
+    }`
+    let demo = loadSource(source, 'api.spec.ts') as any;
+    let test = new demo.Test();
+    t.is(test.check, "Test");
+
+    let errorPath = path.join(__dirname, "resources/error.slint");
+
+    const error = t.throws(() => {
+        loadFile(errorPath)
```

Comment:
Perhaps you meant to use loadSource here? :)

Reply:
ohhh, my oversight. thanks ;)